llvm/test/Analysis/ScalarEvolution/trip-count.ll

```diff
@@ -211,3 +211,217 @@ for.body:
 exit:
   ret void
 }
+
+define void @epilogue(i64 %count) {
```

Comment:
It would probably make sense to move the tests to a separate file, perhaps `max-backedge-taken-count-guard-info-with-multiple-predecessors.ll`

Reply:
thanks, moved as suggested